src/containers/AdminCampaignEdit/sections/CampaignContactsForm/index.tsx

```diff
@@ -140,7 +140,7 @@ class CampaignContactsForm extends React.Component<
         excludeCampaignIds: selectedCampaignIds
       };
       const response = await this.props.mutations.editCampaign(campaignInput);
-      if (response.errors) throw response.errors;
+      if (response.errors) throw response.errors[0];
```

Comment:
Question: Were you able to figure out if this was needed because of changes in #1027 or somewhere else?